drizztsaga/scripts/baf/bg/f_drizzt.baf

```diff
@@ -21,18 +56,21 @@ END
 
 // to prevent immersion-breaking multiple instances of Drizzt's scimitars (if pickpocketed drizzt before he joins)
 IF
-  Delay(10)
-  GlobalLT("F_DrizztVanillaStuffDisposed","LOCALS",3)
+  Delay(1)
+  GlobalGT("F_DrizztVanillaStuffDisposed","LOCALS",0)
```

Comment:
This one is still concerning me... Could be simply `GlobalLT("F_DrizztVanillaStuffDisposed","LOCALS",3)`? I can imagine that another modification could you CHAN06 armor and in this case this script block will fire all the time even if the Drizzt is no longer with a party, right?

Reply:
> his case this script block will fire all the time even if the Drizzt is no longer with a party

It won't as the `F_DrizztVanillaStuffDisposed` is greater than 0 when you have drizzt in your party.

https://github.com/Black-Wyrm-Lair/Drizzt-Saga/blob/750f5aae3bc745858b373611b212e955eab28399/drizztsaga/scripts/baf/bg/f_drizzt.baf#L45-L55

In addition, this block check for drizzt in the party:
https://github.com/Black-Wyrm-Lair/Drizzt-Saga/blob/750f5aae3bc745858b373611b212e955eab28399/drizztsaga/scripts/baf/bg/f_drizzt.baf#L59-L61

I don't like the `GlobalLT("F_DrizztVanillaStuffDisposed","LOCALS",3)` because the block will fire only once. For most of play it should not be an issue but if the player have only one weapon in inventory and one dropped on the ground, the trigger will fire for the one in inventory but not for the second one, when you pick it.

Reply:
Seems reasonable. I will double check this :)